### PR TITLE
upgrade ginkgo and additional feature introspection followup

### PR DESCRIPTION
**Pull request by shajmakh**
This is a followup for https://github.com/openshift-kni/numaresources-operator/pull/958, please see commits for more details.

### CODE REVIEW

**Review by ffromani — COMMENTED** [review]
the only thing I don't like too much is the topic `auto_sched_update_on_kc_change` but I don't have better suggestions anyway

**Comment by shajmakh** [comment]
/hold
@ffromani, if you prefer, including a bug ID for the latest version is also an option; what do you think?

File: internal/api/features/_topics.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "taint",
     "nodelabel",
     "byres",
-    "tmpol"
+    "tmpol",
+    "auto_sched_update_on_kc_change"
```
Comment (ffromani) [inline internal/api/features/_topics.json:19]: maybe `schedattrwatch`?

**Review by ffromani — COMMENTED** [review]
I think topics should preferably be different than bug ids because we talk about something approaching more features than bufixes, but I'm not strongly against using  them.

File: internal/api/features/_topics.json (hunk 1 above)
Comment (shajmakh) [reply internal/api/features/_topics.json:19]: sounds better, will use that

**Review by ffromani — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**